custom_typing/NodePath.ts

```diff
@@ -0,0 +1,32 @@
+interface NodeKey extends Node {
```

Comment:
I have always worked with prefixing the interfaces with capital `I`. What should we do here? I would follow the same pattern